src/client/components/app-header/app-header.vue

```diff
@@ -40,23 +41,54 @@ export default {
 <style>
 @import '../app-core/index.css';
 
-.app-header__menu-link {
+.app-header {
+  width: 100%;
+  height: 80px;
```

Comment:
Since the header is `fixed`/`sticky` the content following it will be under the header. We can fix this by parametrising the header's height:
```css
:root {
  --app-header-height: 80px;
}
.app-header {
  height: var(--app-header-height);
}
.app-header + * { /* whatever element comes directly after the header */
  margin-top: var(--app-header-height);
}
```
I admit, that's not very BEM, but I don't know what other way to solve it.